### PR TITLE
Update us_stirr.m3u

### PR DESCRIPTION
**Pull request by what-a**
Improving some logos.

### CODE REVIEW

File: channels/us_stirr.m3u [hunk 1]
```diff
@@ -351,61 +351,61 @@ https://dai.google.com/linear/hls/event/EpqgwRlpQKq73ySVSohJWA/master.m3u8
...
 https://dai.google.com/linear/hls/event/_PDxBUttQYqkxPnmh3VOZA/master.m3u8
-#EXTINF:-1 tvg-id="STIRRSpace1999.us" tvg-country="US" tvg-language="English" tvg-logo="https://komonews.com/resources/media2/3x1/full/119/center/90/aaaa25c9-d98a-46b8-8dca-3851ab93d3ee-small3x1_STIRR_Logo_0320_SPACE1999_1920x1080_EPG.png?cb=c81e728d9d4c2f636f067f89cc14862c" group-title="",STIRR Space 1999 (768p)
+#EXTINF:-1 tvg-id="STIRRSpace1999.us" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/fr/b/b3/Cosmos_1999.jpg" group-title="",STIRR Space:1999 (768p)
 https://dai.google.com/linear/hls/event/NeKNJHuzSeCiN_7Fcuo83Q/master.m3u8
 #EXTINF:-1 tvg-id="STIRRSports.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/SAfy1us.png" group-title="Sports",STIRR Sports (720p) [Offline]
 https://dai.google.com/linear/hls/event/1B2yihdIR1mCL63rXzERag/master.m3u8
```
Comment (LaneSh4d0w) [inline channels/us_stirr.m3u:411]: If you have an alt to this one, since it's offline, it can be interesting.

**Review by BellezaEmporium — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
Changed some resolutions based on the Google DAI manifest.

**Review by BellezaEmporium — APPROVED** [review]

File: channels/us_stirr.m3u (hunk 1 above)
Comment (what-a) [reply channels/us_stirr.m3u:411]: Which is offline? The Space:1999 stream and suggested logo both appear online for me.

Comment (LaneSh4d0w) [reply channels/us_stirr.m3u:411]: The Stirr Sports one

Comment (what-a) [reply channels/us_stirr.m3u:411]: It seems the "STIRR Sports" channel is gone for good. It's no longer part of the official listing at stirr.com nor a part of Matt Huisman's listing for STIRR.

**Review by MapGuy11 — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**